src/services/user.service.ts

```diff
@@ -31,3 +34,13 @@ export async function createUser(user: User): Promise<User> {
     },
   })) as User;
 }
+
+export async function isPasswordValid(user: User, password: string) {
```

Comment:
Please add return value.

Reply:
For the `createUser` function? Or for the `isPasswordValid`?

The first one should be returning the created user (I think) at the moment. The second one returns `true` or `false`.

Reply:
For `isPasswordValid` should be `isPasswordValid(user: User, password: string): boolean`

Reply:
Fixed.